sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -91,6 +91,7 @@ BOOL WINAPI SHExpandEnvironmentStringsForUserW(HANDLE, LPCWSTR, LPWSTR, DWORD);
 
 
 BOOL WINAPI SHIsEmptyStream(IStream*);
+HRESULT WINAPI IStream_Size(IStream *lpStream, ULARGE_INTEGER* lpulSize);
```

Comment:
I will add the missing IStream_* functions in shlwapi.h in a separate PR.

Reply:
Now or future?

Reply:
later today.